contracts/chain-adapters/Solana_Adapter.sol

```diff
@@ -113,7 +113,7 @@ contract Solana_Adapter is AdapterInterface, CircleCCTPAdapter {
         }
 
         bytes4 selector = bytes4(message[:4]);
-        if (selector == SpokePoolInterface.setEnableRoute.selector) {
```

Comment:
Note to self: can delete this section if we choose to remove deposit routes in Solana spoke